dmutils/forms/mixins.py

```diff
@@ -54,10 +54,13 @@ class DMFieldMixin:
     '''
     def __init__(self, label=None, validators=None, hint=None, **kwargs):
         self.hint = hint
-        self.question = label
 
         super().__init__(label=label, validators=validators, **kwargs)
 
+    @property
+    def question(self):
+        return self.label
```

Comment:
This should probably be a proxy on `self.label.text` specifically. Using `self.label` will emit the text inside HTML `<label>` tags, which we probably wouldn't want in most cases.

Reply:
Ah yes that makes sense